src/main/scala/scalismo/faces/image/PixelImageOperations.scala

```diff
@@ -204,4 +204,15 @@ object PixelImageOperations {
     }
   }
 
+  /** Mirrors the target at the horizontally */
+  def mirrorHorizontal[Pixel](target: PixelImage[Pixel]): PixelImage[Pixel] = {
+    PixelImage(target.width, target.height, (x, y) => target(x, target.height-y))
```

Comment:
There is a -1 missing in the index calculation. A test would be nice to have in such a case to not introduce such bugs.